api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -286,10 +287,25 @@ message ListPlansResponse {
   repeated Plan plan = 1;
 }
 
+// A channel set defines one or both of the uplink and downlink `RadioDeviceConfiguration`s to be
+// used when executing a pass. A channel set is unique to a specific satellite, and a satellite may
+// have one or more channel sets.
```

Comment:
We could say a bit more about the use case of multiple channel sets.

For example, a satellite will have multiple channel sets if it has multiple radio devices for downlink. In this case, it is necessary to select the configuration that will be used during plan execution.

(Or something along those lines - trying to capture the use case that required channel sets to be part of the API).

Reply:
Thanks for the comment! I've added a common example of TT&C and mission channel sets.